pageserver/src/tenant/timeline/uninit.rs

```diff
@@ -207,7 +244,7 @@ impl TimelineUninitMark {
     }
 }
 
-impl Drop for TimelineUninitMark {
+impl<'t> Drop for TimelineUninitMark<'t> {
```

Comment:
Unless the lifetime is really interesting in a context, one could just:

```suggestion
impl Drop for TimelineUninitMark<'_> {
```

Reply:
Applied in 7f131ad28